articles/search/search-indexer-howto-access-private.md

```diff
@@ -275,7 +275,7 @@ A `202 Accepted` response is returned on success. The process of creating an out
 
 ## 2 - Approve the private endpoint connection
 
-Approval of the private endpoint connection is granted on the Azure PaaS side. It might be automatic if the service consumer has a role assignment on the service provider resource. Otherwise, manual approval is required. For details, see [Manage Azure private endpoints](/azure/private-link/manage-private-endpoint).
+Approval of the private endpoint connection is granted on the Azure PaaS side. If the service consumer has a role assignment on the service provider resource, the approval will be automatic. Otherwise, manual approval is required. For details, see [Manage Azure private endpoints](/azure/private-link/manage-private-endpoint).
```

Comment:
I believe this is accurate. Shared private links are created to request auto-approval. On the private endpoint side for the target service, though, I don't know if there's a way to disable auto-approval even when someone has the proper role. That would be my only concern. Otherwise looks like a good change.

Reply:
It's not mentioned on the [linked page](https://learn.microsoft.com/en-us/azure/private-link/manage-private-endpoint?tabs=manage-private-link-powershell#private-endpoint-connections) either, so I don't think so.
> Automatic: If the service consumer has Azure role-based access control (RBAC) permissions on the service provider resource, the consumer can choose the automatic approval method. When the request reaches the service provider resource, no action is required from the service provider and the connection is automatically approved.

Reply:
Agreed and approved. Thanks for making the update.